llvm/test/tools/sycl-post-link/sc_sym_two_refs.ll

```diff
@@ -7,7 +7,7 @@
 target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
 target triple = "spir64-unknown-unknown-sycldevice"
 
-%"sycl::experimental::spec_constant" = type { i8 }
+%"sycl::ONEAPI::experimental::spec_constant" = type { i8 }
```

Comment:
```suggestion
%"spec_constant" = type { i8 }
```
Matching runtime type name is not necessary for this, so I think it makes sense to make them different to avoid "false dependency".